sdk/storage/azure-storage-files-shares/sample/file_share_getting_started.cpp

```diff
@@ -39,7 +39,7 @@ void FileShareGettingStarted()
   {
     std::cout << metadata.first << ":" << metadata.second << std::endl;
   }
-  fileContent.resize(static_cast<std::size_t>(properties.FileSize));
+  fileContent.resize(static_cast<size_t>(properties.FileSize));
```

Comment:
How is this cast safe to do?

Reply:
It's not safe, but this is just sample code

Reply:
We'd want customer facing samples to be production ready/following expected usage patterns, since folks use these as a starting point by copy/pasting.